src/Templates/Server/ServerSettings.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Text;
+using Stl.DependencyInjection;
+
+namespace Samples.Blazor.Server
+{
+    [Settings("Server")]
+    public class ServerSettings
+    {
+        public string GitHubClientId { get; set; } = "7a38bc415f7e1200fee2";
```

Comment:
Put "YOUR_OWN_GITHUB_CLIENT_ID" string instead.

Reply:
Actually, let's leave this client ID & secret for now - just to simplify life of these who'll be testing this.